src/docs/sdk/performance/opentelemetry.mdx

```diff
@@ -18,7 +18,220 @@ When Sentry performance monitoring was initially introduced, OpenTelemetry was i
 
 ## Approach
 
-TODO: Talk about the approach we are using, based on Matt's hackweek project - https://github.com/getsentry/sentry-ruby/pull/1876
+To transform SDK data to Sentry, we rely on the [OpenTelemetry SpanProcessor](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/sdk.md#span-processor). The SpanProcessor is a component that runs on the same thread as the application code, so it effectively runs in process alongside user code and the Sentry SDK.
+
+### Step 1: Implement the SentrySpanProcessor on your SDK
+
+Below we have written a custom SpanProcessor that will transform the OpenTelemetry spans to Sentry spans.
+
+This relies on a couple key assumptions.
+
+1. Hub/Scope propogation works properly in your platform. This means that the hub used in `onStart` is the same hub used in `onEnd` and it means that hubs fork properly in async contexts.
+2. The SDK is initialized before the OpenTelemetry SDK is initialized.
```

Comment:
The otel auto instrumentation in Python relies on starting an agent to run the python code: https://opentelemetry.io/docs/instrumentation/python/automatic/#execute-an-automatically-instrumented-server

Don't know if the initialization order applies to this case.

Reply:
In this case, autoinstrumentation still uses a tracer that gets set up (https://github.com/open-telemetry/opentelemetry-python/blob/main/docs/examples/auto-instrumentation/server_instrumented.py), so users just have to make sure that Sentry is initialized before that file is evaluated.